### ISSUE TITLE
Benchmark Proteus on AI/ML codes 2

### ISSUE THREAD

**Issue by tbennun** [Olympus-HPC/proteus#88]
- [x]  Bug fixes
    - [x] Crash observed in LBANN [softmax](https://github.com/LLNL/lbann/blob/develop/src/layers/activations/softmax.cu)

- [x] Select ML benchmarks to run:
    - [x] Same benchmarks from iteration 1 but in PyTorch
    - [x] ~~GAT (graph neural network with dynamic input sizes)~~ phase 3
- [ ] Inspect performance differences in layer normalization
- [ ] Investigate how PyTorch should be approached (direct, llvm specialization of compiled models, llvm specialization of jitted kernels)
- [ ] Apply proteus annotations to select operators in LBANN and PyTorch
    - [ ] PyTorch [softmax](https://github.com/pytorch/pytorch/blob/c81d4fd0a8ca826c165fe15f83398dbc4e20b523/aten/src/ATen/native/cuda/PersistentSoftmax.cuh#L68)
    - [ ] PyTorch generic elementwise [vectorized](https://github.com/pytorch/pytorch/blob/c81d4fd0a8ca826c165fe15f83398dbc4e20b523/aten/src/ATen/native/cuda/CUDALoops.cuh#L57) / [nonvectorized]
